ecr/object.ecr

```diff
@@ -5,18 +5,20 @@
 
 module <%= namespace_name %>
   <% render_doc(object) -%>
-  @[GObject::GeneratedWrapper]
+  @[GICrystal::GeneratedWrapper]
   class <%= type_name %> <%= parent_class %>
     <% object.interfaces.each do |interface| -%>
     include <%= to_crystal_type(interface, namespace != interface.namespace) %>
     <% end %>
 
+    <% if object.parent.nil? %>
```

Comment:
There is already a `<% if object.parent.nil? %>` in this file. Wouldn't it be better to move this pointer there?

Reply:
The bad side is that the `@pointer` declaration would not be at the top of the class declaration, something that is a "implicit standard". Generate nice, readable and easy to debug code was also one of the motivations to create this shard. I think these two lines here wont hurt in favor of a nicer generated code.